### PR TITLE
Fixes a horrifying brainfart in the body temperature damage calculation

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
rip

### CODE REVIEW

**Comment by vuonojenmustaturska** [comment]
@AnturK

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -295,7 +295,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		C.Digitigrade_Leg_Swap(TRUE)
 	for(var/X in inherent_traits)
 		C.remove_trait(X, SPECIES_TRAIT)
-	
+
```
Comment (Time-Green) [inline code/modules/mob/living/carbon/human/species.dm]: how do you fuck up that badly?

**Comment by Kagus** [comment]
I'm still fairly new to this git, and github in general, so I'm a little uncertain about some things...

When exactly is it appropriate to use the "**FUCK**" label in this repository?

**Comment by epochayur** [comment]
when it's time to panic

**Pull request merged**